### DISCUSSION TITLE
Modifying style attribute via hooked state

### DISCUSSION THREAD

**Discussion by jgburford** [idom-team/idom#468] [Question] [answered]
What I'm trying to do is modify (toggle) the background color of a div when it is clicked.

So I have something like

```python
bgcol, set_bgcol = idom.hooks.use_state('red')

return idom.html.div(
    {
        'onClick': lambda e: set_bgcol('blue' if bgcol == 'red' else 'red'), 
        'style': {'background-color': bgcol}
    },
    "ipsum lorem", 
    bgcol
)
```

inside a component.

I have tried this and a few other variations and so far no luck. I put the text of bgcol into the div as well in order to verify that it is in fact changing. It is, but the style attribute of the div tag is not.

What is/would be (if not implemented yet) the preferred way of doing this?

**Comment by rmorshea** [comment, marked as answer]
All names in IDOM are `camelCase` so the style attribute should be `backgroundColor`
